composer.json

```diff
@@ -32,7 +36,7 @@
         "dmore/behat-chrome-extension": "^1.4",
         "drupal/core-dev": "^10",
         "drupal/drupal-extension": "^5@alpha",
-        "palantirnet/the-build": "^4@beta"
+        "palantirnet/the-build": "dev-dev-37-eslint"
```

Comment:
This is temporary.  After this PR is approved we should be able to revert or update.